Source/SpatialGDK/Private/SpatialInterop.cpp

```diff
@@ -899,10 +906,10 @@ void USpatialInterop::GetSingletonActorAndChannel(FString ClassName, AActor*& Ou
 	NetDriver->SingletonActorChannels.Add(SingletonActorClass, TPair<AActor*, USpatialActorChannel*>(OutActor, OutChannel));
 }
 
-void USpatialInterop::UpdateGlobalStateManager(const FString& ClassName, const FEntityId& SingletonEntityId)
+void USpatialInterop::AddSingletonToGSM(const FString& ClassName, const FEntityId& SingletonEntityId)
```

Comment:
This name is more incorrect that saying `Update` (since update can imply you are changing or adding values to the map). Can you create two functions, one which is for updating the Singleton map (singletons being actor classes that are listed in the settings) and one for adding stably named actors. They interact with different parts of the component so I think it's fine to have two functions.

Reply:
UpdateSingletonId